source/common/router/route_config_update_receiver_impl.h

```diff
@@ -34,6 +34,8 @@ class ConfigTraitsImpl : public Rds::ConfigTraits {
   ProtobufMessage::ValidationVisitor& validator_;
 };
 
+using VirtualHostMap = std::map<std::string, envoy::config::route::v3::VirtualHost>;
```

Comment:
this is fine; or maybe you want to sink this inside RouteConfigUpdateReceiverImpl ?

Reply:
I'm new to Envoy code base, what's the common practice?

Reply:
I've seen both: it all depends on whether that nickname is useful outside that class. At first glance I didn't see any need to have it exposed outside the class.

Reply:
Moved